src/main/scala/fs2/Process1.scala

```diff
@@ -104,6 +104,12 @@ object process1 {
   def scan1[F[_],I](f: (I, I) => I)(implicit F: NotNothing[F]): Handle[F,I] => Pull[F,I,Handle[F,I]] =
     Pull.receive1 { case o #: h => scan(o)(f).apply(h) }
 
+  /** Writes the sum of all input elements, or zero if the input is empty. */
+  def sum[F[_],I:Numeric](implicit F: NotNothing[F]): Handle[F,I] => Pull[F,I,Handle[F,I]] = {
```

Comment:
as a user I wouldn't do that though, I'd basically only define and use my own `MonoidalFns`.

Think about it though, I'm not sure what you're planning with plugging in impls for the `fs2.util.*` I can't really think of a way that doesn't involve orphans. I was assuming that Monoid would use exactly the same technique, but need to think further as to how you might achieve it without magic.